chapters/data-analysis.tex

```diff
@@ -90,19 +90,19 @@ \subsection{Organizing your folder structure}
 
 \subsection{Breaking down tasks}
 
-We divide the process of transforming raw datasets to analysis-ready datasets into four steps:
+We divide the process of transforming raw datasets to analysis-ready datasets to research results into four steps:
```

Comment:
```suggestion
We divide the process of transforming raw datasets to analysis-ready datasets into four steps:
```
sentence wasn't grammatically correct as revised

Reply:
Agree the revised sentece was odd, but I liked it. As those four stes takes us from raw data to results, and not raw data to analysis or final data. What about?

"We divide the process of transforming raw datasets to research outputs into four steps:"

Reply:
sure, just minor grammar correction:
"We divide the process of transforming raw datasets **in**to research outputs into four steps:"